src/libs/SidebarUtils.ts

```diff
@@ -458,7 +442,111 @@ function getOptionData({
     return result;
 }
 
+function getWelcomeMessage(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>): WelcomeMessage {
+    const welcomeMessage: WelcomeMessage = {showReportName: true};
+    if (ReportUtils.isChatThread(report) || ReportUtils.isTaskReport(report)) {
+        return welcomeMessage;
+    }
```

Comment:
Where is this logic coming from?

Reply:
For these cases, I think it doesn't make sense to display the message `This is the beginning of your chat with ` in LHN for the chat thread or task report.